Source/VisualStyle.swift

```diff
@@ -86,12 +86,11 @@ extension VisualStyle {
     public var actionViewSeparatorColor: UIColor { return UIColor(white: 0.5, alpha: 0.5) }
     public var actionViewSeparatorThickness: CGFloat { return 1 / UIScreen.mainScreen().scale }
 
-    public func textColor(forAction action: AlertAction?) -> UIColor {
+    public func textColor(forAction action: AlertAction?) -> UIColor? {
         if action?.style == .Destructive {
             return UIColor.redColor()
-        } else {
-            return UIView().tintColor
         }
+        return nil
```

Comment:
I'd write this as

`return action?.style == .Destructive ? UIColor.redColor() : nil`